PlayCover/View/PlayCoverApp.swift

```diff
@@ -31,7 +31,6 @@ class AppDelegate: NSObject, NSApplicationDelegate {
 		UserDefaults.standard.register(
 			defaults: ["NSApplicationCrashOnExceptions" : true]
 		)
-        LaunchServicesWrapper.setMyselfAsDefaultApplicationForFileExtension("ipa")
```

Comment:
Doesn't it need this to set itself as a default for ipa files, or is it done somewhere else?

Reply:
It was Candy's opinion that setting PlayCover to open `.ipa`s by default without alerting the user isn't great practice. Elsewhere in the `Info.plist` it specifies that PlayCover can open `.ipa` files so it should still be available as an option, but just wont be set as default.

Reply:
Ah, ok. Makes sense.